MOBILE_CLIENT/COMMON/SECURE/SECURE_VIRUSTOTAL_SCAN/description.md

```diff
@@ -0,0 +1 @@
+Performing malware analysis through multiple antivirus scans using VirusTotal's MD5 hash-based search.
```

Comment:
This should tell the user that the results of the analysis are negative

Reply:
+1